table/tables/tables.go

```diff
@@ -104,7 +103,6 @@ func NewTable(tableID int64, tableName string, cols []*column.Col, alloc autoid.
 		indexPrefix:  string(genTableIndexPrefix(tableID)),
 		alloc:        alloc,
 		Columns:      cols,
-		state:        model.StatePublic,
```

Comment:
`meta` is the table schema information.
The table instance should keep a reference to it.

If we have the reference, we don't need to copy the field value in meta.